jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jcmd.java

```diff
@@ -79,19 +79,23 @@ public static void main(String[] args) {
 		} else if ((args.length == 1) && (null != firstArgument) && Arrays.stream(HELP_OPTIONS).anyMatch(firstArgument::equals)) {	
 			System.out.printf(HELPTEXT);
 		} else {
-			String vmid = firstArgument;
-			AttacherDiagnosticsProvider diagProvider = new AttacherDiagnosticsProvider();
 			command = DiagnosticUtils.makeJcmdCommand(args, 1);
-			try {
-				diagProvider.attach(vmid);
+			if (command.isEmpty()) {
+				System.out.printf(HELPTEXT);
```

Comment:
This can print an Error before printing the help output so the User can understand what went wrong.

Reply:
Added an error message, please have another look.

This PR depends on the test fix in https://github.com/eclipse/openj9/pull/6790, otherwise the PR build will fail. (it is due to the help doc change in https://github.com/eclipse/openj9/pull/6789).